evergreen/config.yml

```diff
@@ -790,7 +790,7 @@ tasks:
       script: |-
         export UNITTEST_PROGRESS=1
         export UNITTEST_THREADS=1
```

Comment:
Transactions_ConcurrentFrozenTableGetByName spawns 1000 threads, it's surprising that it wasn't a problem before. 
Valgrind serializes execution of all these threads. Are you sure it makes sense to run it as is? This test is already one of the slowests under valgrind, maybe just tweak test's parameters?